From 11dd1c84879e476a4d3cb264d97bfea9bcfb30d6 Mon Sep 17 00:00:00 2001
From: Dmitry Marakasov <amdmi3@amdmi3.ru>
Date: Tue, 26 May 2020 17:58:59 +0300
Subject: [PATCH] Remove broken mention of testsuite from setup.py
Origin: https://github.com/PyCQA/pycodestyle/commit/11dd1c84879e476a4d3cb264d97bfea9bcfb30d6

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

--- a/setup.py
+++ b/setup.py
@@ -63,5 +63,4 @@
         'Programming Language :: Python :: Implementation :: PyPy',
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
-    test_suite='testsuite.test_all.suite',
 )
